.webpack.config

```
@@ -167,7 +167,13 @@ module.exports = {
                                 ]
                             };
 
-                            const result = optimize(fileContentsString, config);
+                            const result = optimize(fileContentsString, {
+                                ...config,
+                                js2svg: {
+                                    pretty: true,
+                                    encodeEntity: (char) => char  // Don't encode entities
```

```suggestion
                                    // NOTE: disable default character encoding and use characters *as-is*
                                    encodeEntity: (char) => char
```
or add a note in the commit msg instead.